pkg/handlers/v1/ec2.go

```diff
@@ -69,40 +78,113 @@ type ec2ChangedPropsARN struct {
 
 type ec2Transformer struct{}
 
-func (t ec2Transformer) Create(event awsConfigEvent) (Output, error) {
-	output, err := getBaseOutput(event.ConfigurationItem)
+func (t ec2Transformer) Create(event awsConfigEvent) ([]Output, error) {
+	outputs := []Output{}
+	baseOutput, err := getBaseOutput(event.ConfigurationItem)
 	if err != nil {
-		return Output{}, err
+		return []Output{}, err
 	}
-
 	// if a resource is created for the first time, there is no diff.
 	// just read the configuration
 	var config ec2Configuration
 	if err := json.Unmarshal(event.ConfigurationItem.Configuration, &config); err != nil {
-		return Output{}, err
+		return []Output{}, err
 	}
-	change := extractEC2NetworkInfo(&config)
-	change.ChangeType = added
-	output.Changes = append(output.Changes, change)
-	return output, nil
+
+	hasEni := false
+	for _, ni := range config.NetworkInterfaces {
+		ni := ni
```

Comment:
Is there a reason for this line? It seems repetitive here.

Reply:
That was to prevent a lint error that disallows making a reference of a for loop value (&ni)
- [ ] Change this to use the array index instead